### PR TITLE
Code coverage

### PR DESCRIPTION
**Pull request by noloman**
Simple test coverage added

### CODE REVIEW

**Comment by noloman** [comment]
screenshot 2018-11-28 at 16 58 27

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require 'simplecov'
+SimpleCov.start
```
Comment (BerkhanBerkdemir) [inline spec/spec_helper.rb]: Shouldn't we add `'rails'` [like that](https://github.com/colszowka/simplecov#use-it-with-any-framework) according to SimpleCov documentation?

File: .gitignore [hunk 2]
```diff
@@ -27,8 +27,11 @@
...
 /config/master.key
+
+coverage
+
 /public/packs
 /public/packs-test
 /node_modules
 yarn-debug.log*
-.yarn-integrity
+.yarn-integrity
```
Comment (BerkhanBerkdemir) [inline .gitignore]: Just FYI: we use some POSIX standards like a newline end of the file.

File: spec/spec_helper.rb (hunk 1 above)
Comment (noloman) [reply spec/spec_helper.rb]: good catch! nevertheless, it works, but I wonder why 👍

**Pull request merged**